src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DhGaugeParameters.hpp

```diff
@@ -0,0 +1,153 @@
+#include "Options/Options.hpp"
+
+namespace GeneralizedHarmonic::gauges {
+/// Option tags for initializing damped harmonic gauge
+namespace OptionTags {
+/// Gauge-related option tags for the GeneralizedHarmonic evolution system.
+struct GaugeGroup {
+  static std::string name() noexcept { return "Gauge"; }
+  static constexpr Options::String help{
+      "Gauge-specific options for the GH evolution system"};
+  using group = GeneralizedHarmonic::OptionTags::Group;
+};
+
+/// The rollon start time
+struct DampedHarmonicRollOnStart {
+  using type = double;
+  static std::string name() noexcept { return "RollOnStartTime"; }
+  static constexpr Options::String help{
+      "Simulation time to start rolling on the damped harmonic gauge"};
+  using group = gauges::OptionTags::GaugeGroup;
+};
+
+/// The width of the Gaussian for the gauge rollon
+struct DampedHarmonicRollOnWindow {
+  using type = double;
+  static std::string name() noexcept { return "RollOnTimeWindow"; }
+  static constexpr Options::String help{
+      "The width of the Gaussian that controls how quickly the gauge is "
+      "rolled on."};
+  using group = gauges::OptionTags::GaugeGroup;
+};
+
+/// The width of the Gaussian for the spatial decay of the damped harmonic
+/// gauge.
+template <typename Frame>
+struct DampedHarmonicSpatialDecayWidth {
+  using type = double;
+  static std::string name() noexcept { return "SpatialDecayWidth"; }
+  static constexpr Options::String help{
+      "Spatial width of weight function used in the damped harmonic "
+      "gauge."};
+  using group = gauges::OptionTags::GaugeGroup;
+};
+
+/// The amplitudes for the L1, L2, and S terms, respectively, for the damped
+/// harmonic gauge.
+template <typename Frame>
+struct DampedHarmonicAmplitudes {
+  using type = std::array<double, 3>;
+  static std::string name() noexcept { return "Amplitudes"; }
+  static constexpr Options::String help{
+      "Amplitudes [AL1, AL2, AS] for the damped harmonic gauge."};
+  using group = gauges::OptionTags::GaugeGroup;
+};
+
+/// The exponents for the L1, L2, and S terms, respectively, for the damped
+/// harmonic gauge.
+template <typename Frame>
+struct DampedHarmonicExponents {
+  using type = std::array<int, 3>;
+  static std::string name() noexcept { return "Exponents"; }
+  static constexpr Options::String help{
+      "Exponents [eL1, eL2, eS] for the damped harmonic gauge."};
+  using group = gauges::OptionTags::GaugeGroup;
+};
+}  // namespace OptionTags
+
+/*!
+ *  A struct holding the parameters for initializing damped harmonic gauge
+ *  If UseRollon is true, the gauge transitions ("rolls on") to damped harmonic
+ *  fromdetermines whether the gauge transitions to damped harmonic
+ * ("rolls on") from the initial data gauge; otherwise, the gauge begins
+ * immediately in damped harmonic gauge.
+ */
+template <typename Frame, bool UseRollon>
+struct DhGaugeParameters;
+
+template <typename Frame>
```

Comment:
Is it perhaps possible to omit the `Frame` template parameter? It seems like only the compute tag needs the frame in order to retrieve the coordinates from the box, while the options seem to be frame-independent.